src/Network/Send.pm

```diff
@@ -321,6 +321,7 @@ sub reconstruct_master_login {
 
 	$args->{ip} = '192.168.0.2' unless exists $args->{ip}; # gibberish
 	$args->{mac} = $config{macAddress} || '111111111111' unless exists $args->{mac}; # gibberish
```

Comment:
What about:
```perl
	$args->{mac} = uc($config{macAddress}) || '111111111111' unless exists $args->{mac}; # gibberish
```

Reply:
i need to test, idk what perl return if we use uc in a not defined var